Codegen/test/CodegenTestUtils.cs

```diff
@@ -58,6 +58,55 @@ public class CodegenTestUtils {
             TypeCobol.Test.TestUtils.compareLines(path, writer.ToString(), expected, PlatformUtils.GetPathForProjectFile(Path.Combine(ROOT, OUTPUT, path), "Codegen\\Test"));
         }
 
+        /// <summary>
+        /// Parse and generate using DocumentFormat.RDZReferenceFormat by default, because it's our only real target 
+        /// for now and we don't have specifications for FreeFormat.
+        /// </summary>
+        /// <param name="path"></param>
+        /// <param name="skeletons"></param>
+        /// <param name="autoRemarks"></param>
+        /// <param name="copies"></param>        
+        public static void ParseGenerateCompareWithLineMapping(string path, List<Skeleton> skeletons = null, bool autoRemarks = false, string typeCobolVersion = null, IList<string> copies = null)
+        {
+            ParseGenerateCompareWithLineMapping(path, skeletons, DocumentFormat.RDZReferenceFormat, typeCobolVersion, autoRemarks, copies);
+        }
+
+        private static string ML_SUFFIX = ".lm";
+        public static void ParseGenerateCompareWithLineMapping(string path, List<Skeleton> skeletons, DocumentFormat format, string typeCobolVersion, bool autoRemarks = false, IList<string> copies = null)
```

Comment:
Almost all the code is duplicated with method ParseGenerateCompare, can you factorize ?